package Tasks;

import java.util.Scanner;

public class ArrayDuplicateelements {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("enter array size:");
		int n = sc.nextInt();
		
		int []a=new int[n];
		System.out.println("enter array elements:");
		for(int i=0;i<n;i++) {
			a[i]=sc.nextInt();
			
		}
		System.out.println(" unique elements:");
		
		for(int i=0;i<n;i++) {
			boolean duplicate = false;
			for(int j=0;j<i;j++) {
				if(a[i]==a[j]) {
					duplicate = true;
					break;
					
				}
			}
			if(!duplicate) {
				System.out.println(a[i]+" ");
			}
		}
		
	}

}
